packages/gamepads/example/pubspec.yaml

```diff
@@ -10,7 +10,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  gamepads: ^0.1.4
+  flutter_web_plugins:
```

Comment:
Been a while since I developed federated plugins, but do we need this to be added by users in their applications? Can't we add this in the pubspec of the web implementation of the plugin instead?

Reply:
Yes, this can be removed. my mistake.